src/main/resources/network/brightspots/rcv/GuiConfigLayout.fxml

```diff
@@ -312,55 +313,84 @@
-                  </padding>
-                </HBox>
-                <Separator>
-                  <padding>
-                    <Insets bottom="8.0" top="8.0"/>
-                  </padding>
-                </Separator>
-                <HBox alignment="CENTER_LEFT" spacing="4.0">
-                  <Label text="Name *" prefWidth="50.0"/>
-                  <TextField prefHeight="25.0" prefWidth="300.0" fx:id="textFieldCandidateName"/>
-                  <padding>
-                    <Insets bottom="4.0" left="4.0" right="4.0" top="4.0"/>
-                  </padding>
-                </HBox>
-                <HBox alignment="CENTER_LEFT" spacing="4.0">
-                  <Label text="Code" prefWidth="50.0"/>
-                  <TextField prefHeight="25.0" prefWidth="100.0" fx:id="textFieldCandidateCode"/>
-                  <padding>
-                    <Insets bottom="4.0" left="4.0" right="4.0" top="4.0"/>
-                  </padding>
-                </HBox>
-                <HBox alignment="CENTER_LEFT" spacing="4.0">
-                  <CheckBox fx:id="checkBoxCandidateExcluded" mnemonicParsing="false"
-                    text="Excluded">
-                    <VBox.margin>
-                      <Insets top="10.0"/>
-                    </VBox.margin>
-                  </CheckBox>
+            <VBox spacing="20.0">
+              <HBox alignment="CENTER" spacing="75.0">
+                <VBox styleClass="bordered-box" maxWidth="300.0">
+                  <VBox alignment="TOP_CENTER">
+                    <padding>
+                      <Insets left="8.0" right="8.0" bottom="8.0" top="8.0"/>
+                    </padding>
+                    <Label text="Auto-Load Candidates From CVRs" style="-fx-font-weight: bold"/>
+                    <Text
+                        text="Search through all CVRs and load each unique candidate name found."
+                        wrappingWidth="250"
+                    />
+                    <Separator>
+                      <padding>
+                        <Insets bottom="8.0" top="8.0"/>
+                      </padding>
+                    </Separator>
+                    <Region prefHeight="25.0" VBox.vgrow="ALWAYS"/>
+                    <HBox alignment="CENTER" spacing="4.0">
+                      <Button mnemonicParsing="false" onAction="#buttonAutoLoadCandidatesClicked" text="Auto-Load"/>
+                    </HBox>
+                  </VBox>
+                </VBox>
+                <VBox styleClass="bordered-box" maxWidth="400.0">
+                  <VBox alignment="TOP_CENTER">
+                    <padding>
+                      <Insets left="8.0" right="8.0" bottom="8.0" top="8.0"/>
+                    </padding>
+                    <Label text="Manually Add Candidate" style="-fx-font-weight: bold"/>
+                    <Text
+                      text="You can manually add candidates if auto-loading fails to produce acceptable results."
```

Comment:
Under what circumstances could auto-loading not produce the desired outcome? Is that specifically the case in which they want to map multiple strings (using aliases, once we've killed codes) to the same candidate? I guess ideally we would support that somehow, but obviously that's another layer of UX complexity on top of what you've added here.

Reply:
Yes, and perhaps cases where you want non-declared candidates to be listed as Write-Ins, or if `UnrecognizedCandidatesException` isn't implemented in some readers (which it's not required to be).